src/main/java/com/stripe/model/Payout.java

```diff
@@ -1,190 +1,340 @@
 
   /**
-   * Create a payout.
+   * Returns a list of existing payouts sent to third-party bank accounts or that Stripe has sent
+   * you. The payouts are returned in sorted order, with the most recently created payouts appearing
+   * first.
    */
-  public static Payout create(Map<String, Object> params, RequestOptions options)
+  public static PayoutCollection list(Map<String, Object> params) throws StripeException {
+    return list(params, (RequestOptions) null);
+  }
+
+  /**
+   * Returns a list of existing payouts sent to third-party bank accounts or that Stripe has sent
+   * you. The payouts are returned in sorted order, with the most recently created payouts appearing
+   * first.
+   */
+  public static PayoutCollection list(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return request(RequestMethod.POST, classUrl(Payout.class), params, Payout.class, options);
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/payouts");
+    return requestCollection(url, params, PayoutCollection.class, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="list">
   /**
-   * List all payouts.
+   * To send funds to your own bank account, you create a new payout object. Your <a
+   * href="#balance">Stripe balance</a> must be able to cover the payout amount, or you’ll receive
+   * an “Insufficient Funds” error.
+   *
+   * <p>If your API key is in test mode, money won’t actually be sent, though everything else will
+   * occur as if in live mode.
+   *
+   * <p>If you are creating a manual payout on a Stripe account that uses multiple payment source
+   * types, you’ll need to specify the source type balance that the payout should draw from. The <a
+   * href="#balance_object">balance object</a> details available and pending amounts by source type.
    */
-  public static PayoutCollection list(Map<String, Object> params) throws StripeException {
-    return list(params, null);
+  public static Payout create(Map<String, Object> params) throws StripeException {
+    return create(params, (RequestOptions) null);
   }
 
   /**
-   * List all payouts.
+   * To send funds to your own bank account, you create a new payout object. Your <a
+   * href="#balance">Stripe balance</a> must be able to cover the payout amount, or you’ll receive
+   * an “Insufficient Funds” error.
+   *
+   * <p>If your API key is in test mode, money won’t actually be sent, though everything else will
+   * occur as if in live mode.
+   *
+   * <p>If you are creating a manual payout on a Stripe account that uses multiple payment source
+   * types, you’ll need to specify the source type balance that the payout should draw from. The <a
+   * href="#balance_object">balance object</a> details available and pending amounts by source type.
    */
-  public static PayoutCollection list(Map<String, Object> params, RequestOptions options)
+  public static Payout create(Map<String, Object> params, RequestOptions options)
```

Comment:
1) Create (standard): Static method to create a resource

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/Payout.java#L282-L286